rwengine/src/engine/Animator.cpp

```diff
@@ -89,10 +86,8 @@ void Animator::tick(float dt) {
 
 bool Animator::isCompleted(unsigned int slot) const {
     if (slot < animations.size()) {
-        return animations[slot].animation
-                   ? animations[slot].time >=
-                         animations[slot].animation->duration
-                   : true;
+        return animations[slot].animation == nullptr || animations[slot].time >=
+                                                        animations[slot].animation->duration;
```

Comment:
This is less readable imo. Should at least contain braces or made into variables.
I think an early out would also be nice:

```
if (animations[slot].animation == nullptr) {
  return true;
}
return animations[slot].time >= animations[slot].animation->duration;
```